From 6c12ffa822ac9ca58ca665d8deebaf1617616dcd Mon Sep 17 00:00:00 2001
From: Liu Ying <victor.liu@nxp.com>
Date: Fri, 2 Jun 2017 16:18:27 +0800
Subject: [PATCH 1904/5242] MLK-15001-1 arm64: fsl-imx8qm.dtsi: Remove
 framebuffer nodes

commit  6d9489ac472d729d6bf963b4cfea652a424eb0a7 from
https://source.codeaurora.org/external/imx/linux-imx.git

No driver works with the framebuffer nodes, so let's remove them.

Signed-off-by: Liu Ying <victor.liu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts  |    8 ---
 arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi      |   72 --------------------
 2 files changed, 80 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts
index ece42a6..238a859 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts
@@ -310,14 +310,6 @@
 	status = "okay";
 };
 
-&framebuffer1 {
-	status = "okay";
-};
-
-&framebuffer3 {
-	status = "okay";
-};
-
 &imxdpu0 {
 	status = "okay";
 };
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
index 7d5a59c..a662b0d 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
@@ -1051,78 +1051,6 @@
 		status = "disabled";
 	};
 
-	framebuffer0: framebuffer@0 {
-		compatible = "imxdpuv1-framebuffer";
-		reg = <0x0 0x56220000 0x0 0x10000>;
-		clocks =
-			<&clk IMX8QM_DC0_PLL0_CLK>,
-			<&clk IMX8QM_DC0_DISP0_CLK>,
-			<&clk IMX8QM_MIPI0_PXL_CLK>,
-			<&clk IMX8QM_MIPI0_BYPASS_CLK>;
-		clock-names = "clk_pll", "clk_disp", "clk_di", "clk_di_bypass";
-		width = <720>;
-		height = <480>;
-		stride = <(720*4)>;
-		format = "b8g8r8a8";
-		power-domains = <&pd_mipi0>;
-		status = "disabled";
-	};
-
-	framebuffer1: framebuffer@1 {
-		compatible = "imxdpuv1-framebuffer";
-		reg = <0x0 0x56240000 0x0 0x10000>;
-		clocks =
-			<&clk IMX8QM_DC0_PLL1_CLK>,
-			<&clk IMX8QM_DC0_DISP1_CLK>,
-			<&clk IMX8QM_LVDS0_PIXEL_CLK>,
-			<&clk IMX8QM_LVDS0_BYPASS_CLK>,
-			<&clk IMX8QM_LVDS0_PHY_CLK>;
-		clock-names = "clk_pll", "clk_disp", "clk_di",
-			"clk_di_bypass", "clk_di_phy";
-		width = <1920>;
-		height = <1080>;
-		stride = <(1920*4)>;
-		format = "b8g8r8a8";
-		power-domains = <&pd_lvds0>;
-		status = "disabled";
-	};
-
-	framebuffer2: framebuffer@2 {
-		compatible = "imxdpuv1-framebuffer";
-		reg = <0x0 0x57220000 0x0 0x10000>;
-		clocks =
-			<&clk IMX8QM_DC1_PLL0_CLK>,
-			<&clk IMX8QM_DC1_DISP0_CLK>,
-			<&clk IMX8QM_MIPI1_PXL_CLK>,
-			<&clk IMX8QM_MIPI1_BYPASS_CLK>;
-		clock-names = "clk_pll", "clk_disp", "clk_di", "clk_di_bypass";
-		width = <720>;
-		height = <480>;
-		stride = <(720*4)>;
-		format = "b8g8r8a8";
-		power-domains = <&pd_mipi1>;
-		status = "disabled";
-	};
-
-	framebuffer3: framebuffer@3 {
-		compatible = "imxdpuv1-framebuffer";
-		reg = <0x0 0x57240000 0x0 0x10000>;
-		clocks =
-			<&clk IMX8QM_DC1_PLL1_CLK>,
-			<&clk IMX8QM_DC1_DISP1_CLK>,
-			<&clk IMX8QM_LVDS1_PIXEL_CLK>,
-			<&clk IMX8QM_LVDS1_BYPASS_CLK>,
-			<&clk IMX8QM_LVDS1_PHY_CLK>;
-		clock-names = "clk_pll", "clk_disp", "clk_di",
-			"clk_di_bypass", "clk_di_phy";
-		width = <1920>;
-		height = <1080>;
-		stride = <(1920*4)>;
-		format = "b8g8r8a8";
-		power-domains = <&pd_lvds1>;
-		status = "disabled";
-	};
-
 	lvds0: lvds@56241000 {
 		compatible = "fsl,imx8qm-lvds";
 		reg = <0x0 0x56241000 0x0 0x1000>;
-- 
1.7.9.5

